integration/test/performancetest/performance_test.go

```diff
@@ -1,59 +1,269 @@
-// go:build linux && integration
+//go:build linux && integration
 // +build linux,integration
 
 package performancetest
 
-import(
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	//"strconv"
+	"sync"
 	"testing"
 	"time"
-	"log"
-	"context"
+
 	"github.com/aws/amazon-cloudwatch-agent/integration/test"
 )
 
 const (
-	configPath = "resources/config.json"
 	configOutputPath = "/opt/aws/amazon-cloudwatch-agent/bin/config.json"
-	agentRuntimeMinutes = 5 //20min desired but 5mins for testing purposes 
+	agentRuntimeMinutes = 5 //20 mins desired but 5 mins for testing purposes
 	DynamoDBDataBase = "CWAPerformanceMetrics"
-
+	testLogNum = "PERFORMANCE_NUMBER_OF_LOGS"
 )
 
+type LogInfo struct {
+	FilePath string `json:"file_path"`
```

Comment:
I don't think we need `json:` annotations here since we are creating structs ourselves down below this. But I could be wrong about that.

Reply:
yeah no that makes sense since we have to marshal it back into json